platform/security/cas/security-cas-tokenvalidator/src/main/java/ddf/security/cas/WebSSOTokenValidator.java

```diff
@@ -14,6 +14,7 @@
 package ddf.security.cas;
 
 import java.nio.charset.Charset;
+import java.util.Base64;
```

Comment:
Looking at the SAML Specs and how java vs apache do the encoding/decoding it would appear they are inconsistent in their use of linefeeds.  I'll write another ticket to update Base64 handling throughout our SAML code.